node/src/blockchain/blockchain_interface.rs

```diff
@@ -2072,4 +2199,88 @@ mod tests {
 
         assert_eq!(check, vec![11, 22, 33, 44, 55, 66, 77])
     }
+
+    #[test]
+    fn non_clandestine_can_fetch_latest_block_number_successfully() {
+        let prepare_params_arc: Arc<Mutex<Vec<(String, Vec<Value>)>>> =
+            Arc::new(Mutex::new(vec![]));
+        let transport = TestTransport::default()
+            .prepare_params(&prepare_params_arc)
+            .send_result(json!("0x1e37066"));
+        let subject = BlockchainInterfaceNonClandestine::new(
+            transport,
+            make_fake_event_loop_handle(),
+            TEST_DEFAULT_CHAIN,
+        );
+
+        match subject.get_block_number() {
+            Ok(latest_block_number) => assert_eq!(latest_block_number, U64::from(31_682_662u64)),
```

Comment:
Could you please interpret this expected number also as a hexadecimal number? The understanding is gonna be easier thanks to the fast comparison of those two values, even though as different data types. 

`U64::from(0x1e37066u64)`

Reply:
+1

Also, the test would be easier to read if you untangled the Assert from the Act.